### DISCUSSION TITLE
Добрый день, возможно вопрос тупой но я не увидел чтобы это уточнялось в статье

### DISCUSSION THREAD

**Discussion by SwarmRA1Nger** [MrZillaGold/VK2Discord#611] [Общее]
Нужно ли при перезапуске компа снова через командную строку прописывать cd C:\Tools\VK2Discord-master и тд?

**Comment by egorprnn** [comment]
Да, можно создать скрипт и автоматически запускать при перезагрузке, почитайте в интернете

**Comment by uruewjjd** [comment]
Можно создать .bat файл и [закинуть его в автозагрузку](https://lumpics.ru/how-to-add-a-batch-file-to-startup-in-windows-10/)
